assets/tests/userInTestGroup.test.ts

```diff
@@ -0,0 +1,26 @@
+import inTestGroup from "../src/userInTestGroup"
+import getTestGroups from "../src/userTestGroups"
+
+jest.mock("userTestGroups", () => ({
+  __esModule: true,
+  default: jest.fn(() => []),
+}))
+
+describe("User Test Groups API", () => {
```

Comment:
**nitpick:** usually the string we use in `describe` is the name of the function / component being tested.